zcash_test_vectors/orchard/asset_id.py

```diff
@@ -5,15 +5,28 @@
 
 import random
 
+from hashlib import blake2b
 from ..orchard.group_hash import group_hash
 from ..output import render_args, render_tv, option
 
 
 def native_asset():
     return group_hash(b"z.cash:Orchard-cv", b"v")
 
-def asset_id(key, description):
-    return group_hash(b"z.cash:Orchard-cv", key + description)
+
+def encode_asset_id(key, description):
+    return b"\x00" + key + description
```

Comment:
separate `\x00` and call it by name (from the zip: version byte)

Reply:
Done.